nw-firestore-schema.ts

```diff
@@ -63,6 +63,33 @@ interface DayOfEvent {
   description: string;
 }
 
+type Team = "Dev" | "Design" | "Marketing" | "Content Writing" | "cmd-f" | "Hackcamp" | "nwHacks" | "Engagement" | "Sponsorship"
```

Comment:
Can we add "Pres" as a team too? Also will it just be an individual has a Team, and Sections Assigned to them?

Reply:
Sure! I have updated the description of the PR with an explanation for the changes, but yes that's the main idea.